script.js

```diff
@@ -0,0 +1,161 @@
+window.addEventListener('load', () => {
+    const form = document.getElementById('form');
+    const fullName = document.getElementById('fullName');
+    const phoneNumber = document.getElementById('phoneNumber');
+    const selectRegion = document.getElementById('selectRegion');
+    const additionalInfo = document.querySelector('.additionalInfo');
+    const agreeChBox = document.getElementById('agreeChBox');
+    const submitBtn = document.getElementById('submitBtn');
+
+
+
+    const selectData = {
+        'center': 'Cherkasy, Dnipro, Kropyvnytskyi, Poltava, Vinnytsia, Zhytomyr, Not in the list',
```

Comment:
can write key without quotes
```suggestion
        center: 'Cherkasy, Dnipro, Kropyvnytskyi, Poltava, Vinnytsia, Zhytomyr, Not in the list',
```